packages/visual-dev/src/components/stories/Icon.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Icon } from '../Icon'
+
+export default {
+  title: "Components / Icon",
+  component: Icon
+}
+
+export const add = () => <Icon icon={"sqh-add"}/>
```

Comment:
Maybe a couple stories for the `size` prop too?

Reply:
Added stories for small, medium, large and custom.